sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -38,8 +38,8 @@ public class SimpleTokenCache {
     public SimpleTokenCache(Supplier<Mono<AccessToken>> tokenSupplier) {
         this.wip = new AtomicReference<>();
         this.tokenSupplier = tokenSupplier;
-        this.shouldRefresh = accessToken -> OffsetDateTime.now()
-            .isAfter(accessToken.getExpiresAt().minus(REFRESH_OFFSET));
+        this.shouldRefresh =
+            accessToken -> OffsetDateTime.now().isAfter(accessToken.getExpiresAt().minus(REFRESH_OFFSET));
```

Comment:
I'm not sure I prefer this style - I tend to like the previous approach when each line starts with a dot.

Reply:
I'll play with this, there are some limitations here and I don't know if this is one of them